src/firebase.js

```diff
@@ -0,0 +1,21 @@
+import { initializeApp } from 'firebase/app';
+import {getDocs, getFirestore} from 'firebase/firestore';
+const firebaseApp=initializeApp({
+
+    apiKey: "AIzaSyBqeQmeu4TyYxzPJE06hFN6xFpmmavFFPA",
+  
```

Comment:
Suggestions : Never use credentials in files. Declare them in .env files and then access it through 
process.env.[VAR_NAME]